components/links/basic/src/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Link from 'react-router/lib/Link'
+import cx from 'classnames'
+
+class LinkBasic extends Component {
+  _renderContent () {
+    const { icon, literal } = this.props
+    return icon && literal ? <span>{icon}{literal}</span> : literal || icon
+  }
+
+  render () {
+    const className = cx('sui-LinkBasic', this.props.className)
+
+    const {
+      disabled,
+      handleClick,
+      rel,
+      target,
+      title,
+      useReactRouterLinks,
+      url
+    } = this.props
+
+    if (disabled) {
+      return (
+        <span
+          className={className}
+          onClick={handleClick}
+          title={title}
+        >
+          {this._renderContent()}
+        </span>
+      )
+    }
+
+    if (useReactRouterLinks) {
+      return (
+        <Link
+          className={className}
+          onClick={handleClick}
+          rel={rel}
+          target={target}
+          title={title}
+          to={url}
+        >
+          {this._renderContent()}
+        </Link>
+      )
+    }
+
+    return (
+      <a
```

Comment:
Missing hreflang, onfocus and onblur, in case someone needs it.

Reply:
I only have just copied the old one. :) I rather prefer people to add them in case they need. Anyway none of the three mentioned make sense for the purposes of this component.